### ISSUE TITLE
GraphQL temporarily 'loses' its schema after an unspecified amount of time

### ISSUE THREAD

**Issue by martaver** [dgraph-io/dgraph#5818]
### Steps to reproduce the issue (command/config used to run Dgraph).

After having a Slash GraphQL endpoint functioning, with a schema and data having been entered into the db, I go away to do some non-nerd  stuff for a while and come back the next day to continue. When I load up my app, the first queries result in:
```
GraphQL error: Not resolving getOrganization. There's no GraphQL schema in Dgraph. Use the /admin API to add a GraphQL schema
```

I check the endpoint in the Slash GraphQL Admin UI, and the schema exists, along with the data.

After waiting a minute or two, the error disappears, and the queries complete correctly again.

### Expected behaviour and actual result.

Queries to Slash GraphQL to work every time, with no exceptions.

I'm assuming that this is something to do with the dgraph instance being cold started in response to a request after a long idle time. Even so, the very first request should already complete successfully.

**Comment by gja** [comment]
Thank you for flagging this. 

The team had identified this issue earlier, and this has been fixed in #5802. This should arrive in Slash GraphQL by the end of this week. 

Sorry for the experience.

**Comment by lgalatin** [comment]
Closing   Fixed in #5802
